example/puppeteer/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "loadgenerator-puppeteer-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx foo.spec.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "puppeteer": "^1.19.0",
+    "mocha": "6.2.0"
```

Comment:
with mocha you're using exact version and with puppeteer not - is there some reason for this?